integration/trafficdumper/dump_streaming_delta.go

```diff
@@ -0,0 +1,53 @@
+package trafficdumper
+
+import (
+	"fmt"
+	"time"
+
+	proxy "github.com/Telenav/osrm-backend/integration/pkg/trafficproxy"
+	"github.com/golang/glog"
+)
+
+// DumpStreamingDelta dumps traffic response from streaming delta channel.
+func DumpStreamingDelta(responseChan <-chan proxy.TrafficResponse) {
+
+	h := New()
+	if h.writeToFile && flags.streamingDeltaSplitDumpFiles {
+		h.updateDumpFileNamePrefix()
```

Comment:
Yes, we can. But I think it's still better to keep the flags in packge, so that callers will automatically have these flags when they import it. It will be easier to use. Then if some times the caller have strongly request to avoid flags like that, we can simply move the flags to main package. How's your opioin? 

For the second question, that's good point I think. I have updated it.